util/string.go

```diff
@@ -304,3 +305,18 @@ func ParseIdToString(input interface{}) (string, error) {
 		return "", fmt.Errorf("unsupported id type: %T", input)
 	}
 }
+
+func ParseFileType(contentType, fileName string) string {
+	if contentType == "" || !strings.Contains(contentType, "/") {
```

Comment:
Use the original code, don't re-implement it

Reply:
@hsluoyz previous commit reverted. Now use appending "/" to avoid panics and try to minimize changes.